salt/kube-apiserver/init.sls

```diff
@@ -0,0 +1,38 @@
+include:
+  - repositories
```

Comment:
It would be better to include kube-common here (and all the other kube-* service roles), and in kube common include `repositories` and `cert`.